pypeit/calibrations.py

```diff
@@ -630,65 +654,48 @@ def get_flats(self):
             self.flatimages = flatfield.FlatImages(None, None, None, None)
             msgs.warn("No pixelflats provided")
```

Comment:
Change this warning to something more appropriate. Can't say what that is yet, as I don't follow your logic here. See above.

Reply:
warning removed.